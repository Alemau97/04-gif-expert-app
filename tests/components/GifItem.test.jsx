import { render } from "@testing-library/react";
import { GifItem } from "../../src/components/GifItem";

describe('Pruebas en <GifItem />', () => { 
    
    test('Debe hacer match con el snapshot', () => { 

        const title = 'Taylor';
        const url = 'https://taylorswift.com';

        const {container} = render(<GifItem  title={title} url={url} />)
        expect(container).toMatchSnapshot();

    });

});