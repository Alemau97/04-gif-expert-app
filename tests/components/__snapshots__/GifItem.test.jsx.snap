// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifItem /> Debe hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Taylor"
      src="https://taylorswift.com"
    />
    <p>
      Taylor
    </p>
  </div>
</div>
`;
